resources/bundles/org.eclipse.core.resources/META-INF/MANIFEST.MF

```diff
@@ -35,3 +35,9 @@ Service-Component: OSGI-INF/ResourceChangeListenerRegistrar.xml,
 Bundle-RequiredExecutionEnvironment: JavaSE-17
 Automatic-Module-Name: org.eclipse.core.resources
 Import-Package: org.eclipse.osgi.service.datalocation;version="[1.3.0,2.0.0)"
+Service-Component: OSGI-INF/org.eclipse.core.internal.resources.CheckMissingNaturesListener.xml
```

Comment:
remove that. strange: i got that automatically added in a PR too

Reply:
This is really strange. I did not include this in a commit. I deleted the lines in 97dadc19c97b4e7dcdc2ac5ac668611fc60886d8 but I needed to do it with the windows text editor because some configuration always added the lines when I safed the file.